scripts/repl.js

```diff
@@ -284,7 +371,34 @@
   /*
    * Initialize the REPL
    */
-  var repl = new REPL();
+  var initialQuery = UriUtils.parseQuery();
+  if (initialQuery && initialQuery.short) {
```

Comment:
Okay, I'll try to do this later today.  It can load the old urls, and just have the url always be `#` or `#?short=...`.